.travis.yml

```diff
@@ -0,0 +1,23 @@
+dist: trusty
+sudo: false
+
+jobs:
+  include:
+    - state: Angular tests
+      language: node_js
+      node_js:
+        - '13'
+      cache:
+        directories:
+          - ./node_modules
+      addons:
+        chrome: stable
+      before-install: cd apsi-enrollment-fe
```

Comment:
maybe better option is to create makefile for whole both back and frontend and run simple make commands without using cd operations

Reply:
Done